helm/snark-worker/templates/snark-worker.yaml

```diff
@@ -32,10 +32,14 @@ spec:
         image: {{ $.Values.mina.image }}
         {{ if $.Values.mina.useCustomEntrypoint -}}
         command: [{{ $.Values.mina.customEntrypoint }}]
-        {{- else }}
```

Comment:
Would this work for non-integration-test deployments as well? cc @QuiteStochastic

Reply:
i THINK george's change should be fine.  the default daemon entrypoint is more or less just `mina $INPUT_ARGS $EXTRA_FLAGS 2>mina-stderr.log 1>mina.log &`.  so if there is no `command`, then args will be passed straight to that.